### ISSUE TITLE
训练开始几个epoch后损失不再下降

### ISSUE THREAD

**Issue by daihz** [zhouhaoyi/Informer2020#394]
作者您好，我是用informer模型应用于自己的数据集，我的数据集是三个变量预测另一个变量，具体为三列长度为300的序列预测一个长度为1的值 每组数据来自不同站点 
在模型训练的过程中发现：不管怎么调整超参数，模型在训练四五个epoch后损失就基本不会再下降，直到早停触发结束训练。
不知道这个现象是否是来自我的超参数设置，还是由于我的数据物理意义与您的数据不同。
以下是我的部分超参数设置：

> Namespace(**activation**='gelu',  **attn**='prob', **batch_size**=50,  **c_out**=1,**seq_len**=300,  **label_len**=300, **dec_in**=3, **enc_in**=3, **n_heads**=8,  **d_model**=64,   **d_ff**=64, **e_layers**=6,  **d_layers**=5, **s_layers**=[3, 2, 1],des='forecasting',  devices='0,1,2,3', distil=True, **dropout**=0.05,  embed='timeF',  **factor**=5, features='S',  freq='w', gpu=0, inverse=False, is_show_label=True, itr=1,  **learning_rate**=0.001, **loss**='mse', **lradj**='type1', **mix**=True, **model**='informerstack',    num_workers=0,   output_attention=True, padding=0,  **patience**=30,  **pred_len**=1,  random_choos=True,  scale=False, seed=12345,shuffle=True, ')

以下为学习曲线
![learning curve](https://user-images.githubusercontent.com/111106962/199740947-05375127-b3b3-4169-a7ea-c2b41d9c7f6b.png)

请问如何解决这种情况，在超参数调优上有没有什么建议，十分感谢

**Comment by hsuxiang** [comment]
哈喽同学，请教一下你找到问题所在了吗？我也是遇到了这个问题，loss在一个很大的值震荡不下降

**Comment by daihz** [comment]
> 哈喽同学，请教一下你找到问题所在了吗？我也是遇到了这个问题，loss在一个很大的值震荡不下降

我把变化的学习率调成固定学习率会好一些，loss会开始下降，但是会造成一些过拟合，效果还在调试。
具体是将EXP_Informer.train()中的更新学习率这句注释掉

> adjust_learning_rate(model_optim, epoch+1, self.args)

**Comment by hsuxiang** [comment]
好的 谢谢你！


发件人: dhz ***@***.***>
发送时间: Wednesday, November 16, 2022 11:06:56 PM
收件人: zhouhaoyi/Informer2020 ***@***.***>
抄送: Hsuxiang ***@***.***>; Comment ***@***.***>
主题: Re: [zhouhaoyi/Informer2020] 训练开始几个epoch后损失不再下降 (Issue #394)


哈喽同学，请教一下你找到问题所在了吗？我也是遇到了这个问题，loss在一个很大的值震荡不下降

我把变化的学习率调成固定学习率会好一些，loss会开始下降，但是会造成一些过拟合，效果还在调试。
具体是将EXP_Informer.train()中的更新学习率这句注释掉

adjust_learning_rate(model_optim, epoch+1, self.args)

―

**Comment by zhouhaoyi** [comment]
根据不同的数据集，可以考虑修改学习率的策略。
